rust/decentraland-godot-lib/src/dcl/js/js_modules/Runtime.js

```diff
@@ -1,19 +1,16 @@
 module.exports.getRealm = async function (body) {
     return {
-        realmInfo: {
-            baseUrl: "https://localhost:8000",
-            realName: "LocalPreview",
-            networkId: 1,
-            commsAdapter: "offline",
-            isPreview: true,
-        }
+        realmInfo: await Deno.core.ops.op_get_realm()
+    }
+}
+module.exports.getWorldTime = async function (body) {
+    const seconds = 60 * 60 * 12 // noon seconds
```

Comment:
add todo, I understand this is hardcoded for now, right?

Reply:
Until we implement a skybox with time, yes. Adding TODO